model/checkin.json

```diff
@@ -0,0 +1,136 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Checkin",
+  "description": "An Elastic Agent checkin to Fleet",
+  "type": "object",
+  "properties": {
+    "_id": {
+      "description": "The unique identifier for the Elastic Agent checkin",
+      "type": "string",
+      "format": "uuid"
+    },
+    "@timestamp": {
+      "description": "Date/time the checkin was created",
+      "type": "string",
+      "format": "date-time"
+    },
+    "agent": {
+      "title": "Checkin Agent",
+      "description": "The current metadata information for the Elastic Agent",
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
Lets use the same structure for agent in all docs. I like this one here ;-)

Reply:
Will do.